tests/integration/conftest.py

```diff
@@ -53,7 +55,7 @@
                                   "WORKSPACE": "WORKSPACE",
                                   "CVS_BRANCH": "CVS_BRANCH",
                                   "GIT_COMMIT": "GIT_COMMIT",
-                                  "GIT_URL": "https://github.com/rcbops/mk8s.git",
+                                  "GIT_URL": "Unknown",
```

Comment:
Setting this to "Unknown" prevents ZigZag from trying to build "Failure Links" which is the default state we want for fixtures.